browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -4067,6 +4067,39 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_BRAVE_HOMEPAGE_MOBILE_BOOKMARKS_TEXT" desc="Text for `Mobile bookmarks` setting in `Homepage` settings">
       Mobile bookmarks
     </message>
+    <message name="IDS_BRAVE_ORIGIN_PREMIUM" desc="Title for Origin Premium">
+      Origin
+    </message>
+    <message name="IDS_YEARLY_SUBSCRIPTION_AMOUNT_ORIGIN" desc="Brave Origin plan text">
+      <ph name="YEARLY_AMOUNT">%1$s</ph> /year
+    </message>
+    <message name="IDS_MONTHLY_SUBSCRIPTION_AMOUNT_ORIGIN" desc="Brave Origin plan text">
+      <ph name="MONTHLY_AMOUNT">%1$s</ph> /month
+    </message>
+    <message name="IDS_ORIGIN_SUBSCRIPTION_TRY_FREE_TRIAL" desc="Try free trial now button for Brave Origin subscription">
+      Try 7 days free
+    </message>
+    <message name="IDS_ORIGIN_PAYWALL_TEXT_1" desc="Brave Origin paywall text">
+      Support Brave's industry-leading work for Web privacy and open-source adblocking lists
+    </message>
+    <message name="IDS_ORIGIN_PAYWALL_TEXT_2" desc="Brave Origin paywall text">
+      Optimized for a more minimalist browser interface
+    </message>
+    <message name="IDS_ORIGIN_PAYWALL_TEXT_3" desc="Brave Origin paywall text">
+      A core of Brave Shields for adblock, privacy, and speed
+    </message>
+    <message name="IDS_ORIGIN_PAYWALL_TEXT_4" desc="Brave Origin paywall text">
+      Support for IT-managed policies at work
+    </message>
```

Comment:
These are confusing the way they are all put together, but won't hold the PR.

Reply:
I think the confusion is because the first item is what you would do if you subscribe to Origin and the rest of the items are the benefits of Origin. Also, I am not sure it's clear what "a core" here means.